src/kbs1.jl

```diff
@@ -0,0 +1,91 @@
+"""
+    test1!(u::Word, v::Word, rs::RewritingSystem, o::Ordering)
+Adds a rule to a rewriting system (if necessary) that insures that there is
+a word derivable form two given words using the rules in rewriting system.
+See [Sims, p. 69].
+"""
+function test1!(u::Word, v::Word, rs::RewritingSystem, o::Ordering)
+    a = rewrite_from_left(u, rs)
+    b = rewrite_from_left(v, rs)
+    if a != b
+        lt(o, a, b) ? push!(rs, b=>a) : push!(rs, a=>b)
+    end
+end
+
+"""
+    overlap1!(i::Integer, j::Integer, rs::RewritingSystem, o::Ordering)
+Checks the overlaps of right sides of rules at position i and j in the rewriting
+system in which rule at i occurs at the beginning of the word. Adds new rules
```

Comment:
the second part of the first sentence is not clear to me

Reply:
`When failures of local confluence are found, new rules are added.` would be better? (verbatim from Sims)

Reply:
i meant the first sentence: `Checks the overlaps of right sides of rules at position i and j in the rewriting system in which rule at i occurs at the beginning of the word`
what is `the word` here?

Reply:
Ahh 😅. `the Word` is the same as `the overlap` from the beginning of the docstring.
```
Checks the overlaps of right sides of rules at position i and j in the rewriting
system in which rule at i occurs at the beginning of the overlap.
```
should be more clear.